lib/x509.js

```diff
@@ -2543,6 +2541,21 @@ pki.getTBSCertificate = function(cert) {
   return tbs;
 };
 
+const jan_1_1950 = new Date('1950-01-01T00:00:00Z');
+const jan_1_2050 = new Date('2050-01-01T00:00:00Z');
+
+function dateToPem(date){
```

Comment:
This is going to ASN1.  Should rename it to `_dateToAsn1` and perhaps move it near one of the other `_*ToAsn1` helper functions.